index.html

```diff
@@ -158,7 +158,7 @@
                         document.getElementById(
                             "loading-image-container"
                         ).innerHTML =
-                            '<img src="loading-animation.gif" style="max-width:90%; width: 1024px; margin: auto; display: block; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);" />';
+                            '<video autoplay loop muted playsinline width="100%" height=auto><source src="loading-animation.webm" type="video/webm"><source src="loading-animation.mp4" type="video/mp4"></video>';
```

Comment:
Is there a reason the movie plays the sequence twice?

Reply:
> Maybe we can reduce the width a bit? Maybe 80%. Otherwise it looks great.

yeah, sure.

Reply:
> Can you please add a .md file with instructions on how you created the video from the artwork?

Yes, I think it will be really useful .

Reply:
> Is there a reason the movie plays the sequence twice?

I converted the GIF directly to .mp4 and .webm to make things robust , I will check if the individual files are playing it twice or both types of files are playing one after another

Reply:
> Maybe we can reduce the width a bit? Maybe 80%. Otherwise it looks great.

I checked with 80% , it was too low so I made it 98%

<img width="960" alt="image" src="https://user-images.githubusercontent.com/75945709/174167040-48f317b7-cd1d-4f1e-94d3-351c14e91d61.png">

Reply:
> Is there a reason the movie plays the sequence twice?

Even after reducing the load time significantly, the duration of video once is not enough time for the page to load completely so inside the video tag I have used "loop" so that we have same experience as of a GIF.
As I go on with my aim to reduce time for loading requirements it might be possible to load the page fully just by playing the video once